systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -439,11 +440,10 @@ void testTlsExternalUser(ExtensionContext extensionContext) throws IOException,
     void setup(ExtensionContext extensionContext) {
         this.clusterOperator = this.clusterOperator
             .defaultInstallation(extensionContext)
+            .withBindingsNamespaces(List.of(Environment.TEST_SUITE_NAMESPACE))
```

Comment:
Should be there also `Constants.CO_NAMESPACE` in the list?

Reply:
No, as in SetupClusterOperatorBuilder defaultInstallation no matter the RBAC. `clusterOperatorBuilder.withNamespace(Constants.CO_NAMESPACE)` is always returned. This PR only makes sure that if we are not deploying resources in setup method directly into co-namespace, the test-suite-namespace is created. Thus covering both namespace and cluster rbac scopes.

Reply:
Okay, in that case it's not needed in all other testsuites where we are using both in the `withBindingsNamespaces` method.

Reply:
Ye this might be cool to do in another PR to clean it up a bit, will do

Reply:
EDIT: we should fix this inside operator installation, as the  cluster.createNamespaces(collectorElement, namespaceInstallTo, bindingsNamespaces); only creates the ones it binds to. So i will at the end add the Constants.CO_NAMESPACE and work on it in another PR. Thanks !